Sky Atlas/Sky Atlas.html

```diff
@@ -28113,6 +28137,160 @@ <h1>Agent Scope Database</h1>
             </code></pre>
           </td>
         </tr>
+        <tr>
+          <td>
+            <dfn>Ethereum Mainnet - Aave USDe Instance Configuration Document</dfn>
```

Comment:
```suggestion
            <dfn>Ethereum Mainnet - Aave aUSDe Instance Configuration Document</dfn>
```

I think this should be aUSDe, if that is correct then there are a couple of instances that must be updated.

Reply:
It was written this way to remain consistent with the USDC instances. Either is fine, but we should update all of them.

We would lean toward using "USDe" instead of "aUSDe" as it makes it references directly the asset being deposited. However, if AA want us to switch them all to `aUSDe` we can.

Reply:
Logically it would follow that we would also need to update all Curve, SparkLend and Morpho instance titles to reference the receipt token instead of the deposit token.

Reply:
Okay, no keep it as you have it then and we can do an edit for all of the relevant protocols later if needed.